vtool/__init__.py

```diff
@@ -7,7 +7,7 @@
 """
 # flake8: noqa
 from __future__ import absolute_import, division, print_function
-__version__ = '2.1.1'
+__version__ = '2.0.0'
```

Comment:
There's already a `v2.0.0` tag so I think it might be better to go with `v3.0.0`?  Or should we go with `v2.2.0`?

Reply:
Changed to 3.0.0 👍